src/components/Homepage/Board/Game/game.css

```diff
@@ -86,4 +107,56 @@ padding:6px 0 6px 0;
 .dice__plateau {
   display: flex;
   flex-direction: column;
-}
\ No newline at end of file
+}
+
+/*--------------------------------------------------------------*/
+/*----------------------------Button CSS------------------------*/
+/*--------------------------------------------------------------*/
+
+.weaponButton {
+  overflow:hidden;
+  -webkit-border-radius:50px;
+  -moz-border-radius:50px;
+  border-radius:50px;
+  width:90px;
+  height:90px;
+  margin: 10px
+}
+
+.styled {
+  border-radius:12px 0 12px 0;
+  background: #f5f5f5;
+  border:none;
+  color:#555;
+  font:bold 12px Verdana;
+  padding:6px 0 6px 0;
+  box-shadow:1px 1px 3px #999;
+  width: 100px;
+  margin: 20px;
+}
+
+
+
+
+
+
+
+.affichageON {
+}
+
+.affichageOFF {
+display: none;
+}
+
+
+
+
+
+
+
+
+
+
+
+
+

```

Comment:
if these classes are used, they should be replaced with conditional rendering. otherwise, they can be removed ;)